gulpfile.js

```diff
@@ -0,0 +1,101 @@
+var _ = require('lodash');
+var gulp = require('gulp');
+var gutil = require('gulp-util');
+var ts = require('gulp-typescript');
+var tslint = require('gulp-tslint');
+var realTslint = require('tslint');
+var spawnSync = require('child_process').spawnSync;
+
+function build(src, dest) {
+  var tsProject = ts.createProject('configs/tsconfig-base.json', { declaration: true });
+  return gulp.src(src)
+    .pipe(tsProject())
+    .pipe(gulp.dest(dest));
+}
+
+gulp.task('build:capnp', function () {
+  return build('./packages/capnp-ts/src/**/*.ts', 'packages/capnp-ts/lib');
+});
+
+gulp.task('build:capnpc', function () {
+  // Stub, schema are not compiled yet
+  return build('./packages/capnpc-ts/src/**/*.ts', 'packages/capnpc-ts/lib');
+});
+
+gulp.task('build', ['build:capnp'/*, 'build:capnpc'*/]);
```

Comment:
You're probably confusing building the `capnpc-ts` library with actually compiling the `.capnp` schema files, which is definitely going to be a separate build task.

The generated schema files should be checked into git (for example, see `packages/capnp-ts/src/std/schema.capnp.ts`), so you'd only want to run something like `gulp capnp-compile` locally and commit the changed files if you make changes to the schema files or the compiler.

You don't need to worry about it right now, but it should be as simple as spawning a shell command via gulp:

```shell
capnpc -o packages/capnpc-ts/bin/capnpc-ts path/to/schema-file.capnp
```